apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -343,12 +345,16 @@ const DanceAiModal: React.FunctionComponent = () => {
     currentToggle === Toggle.CODE &&
     (aiOutput === AiOutput.GENERATED_BLOCKS || aiOutput === AiOutput.BOTH);
 
-  let botImage = aiBotBorder;
+  let aiBotHead = aiBotHeadNormal;
+  let aiBotBody = aiBotBodyNormal;
   if (mode === Mode.GENERATING && generatingProgress.subStep >= 1) {
-    botImage =
-      generatingProgress.step < BAD_GENERATED_RESULTS_COUNT
-        ? aiBotNo
-        : aiBotYes;
+    if (generatingProgress.step < BAD_GENERATED_RESULTS_COUNT) {
+      aiBotHead = aiBotHeadNo;
+      aiBotBody = aiBotBodyNo;
+    } else {
+      aiBotHead = aiBotHeadYes;
+      aiBotBody = aiBotBodyYes;
+    }
```

Comment:
I know it's not happening here, but there's the possibility of the head and body getting out of sync (where Head is set to No and Body is set to Yes).

First question is...do we care? If we want to have a Yes Head and No Body, then we're gtg.

Otherwise, would it be better to bundle them up into an object?

```
const aiBotYes = { head : aiBotHeadYes, body : aiBotBodyYes }
const aiBotNo = { head : aiBotHeadNo, body : aiBotBodyNo }

let aiBot = { head : aiBotHeadNormal, body : aiBotBodyNormal}
...
// and then later
...
aiBot = generatingProgress.step < BAD_GENERATED_RESULTS_COUNT ? aiBotNo : aiBotYes
...
// and then still later
src = {aiBot.head} or src = {aiBot.body}

Reply:
Good question.  There were actually some proposed designs in which we had multiple body expressions but not as many head variants, in which case we'd want to set them independently, so I think it's alright to keep the current architecture in case those additional images do come through.